src/xunit.netcore.extensions/xunit.netcore.extensions.csproj

```diff
@@ -10,6 +10,8 @@
     <RootNamespace>Xunit.NetCore.Extensions</RootNamespace>
     <CLSCompliant>false</CLSCompliant>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
```

Comment:
I'm not sure that is related. Can you follow-up to figure out what that changes in the package or output?